src/EventsPage/components/AllUpcomingEvents.js

```diff
@@ -38,6 +43,6 @@ const renderEventList = (events) => {
 					</div>
 				</Link>
 	 		</div>
-		);
+		)};
```

Comment:
Like the other nitpick, I'd move this brace to its own line (and move the semicolon inside it).